Content/src/Client/Client.fsproj

```diff
@@ -19,17 +15,17 @@
     <ItemGroup>
         <ProjectReference Include="..\Shared\Shared.fsproj" />
     </ItemGroup>
-    <!--#if (minimal)
-  <ItemGroup>
-    <PackageReference Include="Fable.Core" Version="3.1.5" />
-    <PackageReference Include="Fable.Elmish" Version="3.0.6" />
-    <PackageReference Include="Fable.Elmish.React" Version="3.0.1" />
-    <PackageReference Include="Fable.Elmish.Debugger" Version="3.2.0" />
-    <PackageReference Include="Fable.Elmish.Hmr" Version="4.0.1" />
-    <PackageReference Include="Fable.React" Version="5.3.6" />
-    <PackageReference Include="Thoth.Fetch" Version="2.0.0" />
-  </ItemGroup>
-  #else-->
+    <!--#if (minimal)-->
```

Comment:
Is this just indent changes or has the extra --> been added?

Reply:
The extra XML comment syntax just makes it so that you can edit the template file without it breaking syntax highlighting in your editor. This is in line with what was already being done further up in the file.

The only change to this part of the file with the PackageReferences in the output is correcting the indentation.